setup.cfg

```diff
@@ -36,8 +36,8 @@ per-file-ignores =
 
 # pytest
 [tool:pytest]
-addopts = -v -x --xdoc --cov=foreshadow --cov-config=setup.cfg --cov-report=term --cov-report=html -n=8
-;addopts = -s -vv --xdoc
+; addopts = -v -x --xdoc --cov=foreshadow --cov-config=setup.cfg --cov-report=term --cov-report=html -n=8
+addopts = -s -vv --xdoc
```

Comment:
For the dev branch, I believe we shouldn't change these two lines. It's only for running test in PyCharm.

Reply:
My bad, I had to change this to debug test cases. Will change it back.